elliottlib/cli/verify_attached_bugs_cli.py

```diff
@@ -33,24 +34,32 @@ async def verify_attached_bugs_cli(runtime: Runtime, verify_bug_status: bool, ad
     Otherwise, prints the number of bugs in the advisories and exits with success.
     """
     runtime.initialize()
-    advisories = advisories or [a for a in runtime.group_config.get('advisories', {}).values()]
-    if not advisories:
-        red_print("No advisories specified on command line or in group.yml")
+    if advisories:
+        click.echo("WARNING: Cannot verify advisory bug sorting. To verify that bugs are attached to the "
+                   "correct release advisories, run with --assembly=<release>")
+        advisory_id_map = {'?': a for a in advisories}
```

Comment:
In theory, we can get the impetus for the advisory by looking at the first comment, though it might not be necessary.